terraform/environments/bootstrap/delegate-access/iam.tf

```diff
@@ -1,11 +1,10 @@
 locals {
-
-
   account_name = try(regex("^bichard*.|^remote-supervisio*.", terraform.workspace), replace(terraform.workspace, regex("-[^-]*$", terraform.workspace), ""))
-
-
   account_data = jsondecode(file("../../../../environments/${local.account_name}.json"))
+}
 
+data "aws_iam_user" "cicd_member_user" {
```

Comment:
Just thinking...this data look up will fail if we ever rebuild this from scratch as the user won't exist yet.  Would it be possible to get the arn from the module where it is created on line 20 please?  You'll have to add an output for the arn to the module.

Reply:
@davidkelliott - I had considered that as an alternative, yes. I will explore that now. Should have a PR up that covers that change shortly.